src/main.py

```diff
@@ -231,7 +231,7 @@ def run(self):
                                     sub.unsticky()
 
                                 print "Submitting game thread..."
-                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game"))
+                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game",self.TEAM_CODE))

```

Comment:
I changed it to use the home team's code from the get_notes lookup. I thought about trying to determine whether the bot's configured team is home or away, but it didn't seem worth it. It will only matter if one of the two teams is playing a doubleheader against two different teams which never happens (or a spring training split squad, which is already not fully supported).